signbank/dictionary/management/commands/update_wordnet_links.py

```diff
@@ -21,22 +22,36 @@ class Command(BaseCommand):
     def download_wordnet_gloss(self):
         """ Download the WordNet gloss file from the server. """
 
-        DOWNLOAD_URL = "https://www.sign-lang.uni-hamburg.de/easier/sign-wordnet/static/tab/sign_wordnet_gloss_dse.tab"
+        download_url = "https://www.sign-lang.uni-hamburg.de/easier/sign-wordnet/static/tab/sign_wordnet_gloss_dse.tab"
 
-        response = requests.get(DOWNLOAD_URL)
+        response = requests.get(download_url)
         if response.status_code == 200:
-            with open(os.path.join(WRITABLE_FOLDER, "synsets/sign_wordnet_gloss_dse.tab"), "wb") as f:
-                f.write(response.content)
+            glosses_wordnet = response.content
             print("WordNet gloss file downloaded successfully.")
+            return glosses_wordnet
         else:
             print("Failed to download WordNet gloss file. Status code:", response.status_code)
 
+    def download_csv(self, session, download_url, csv_type):
+        """ Download the WordNet links and signs CSV file from the server. """
+
+        download_payload = {
+            "submit": csv_type
+        }
+        download_response = session.post(download_url, data=download_payload)
+        if download_response.status_code == 200:
+            csv = download_response.content
+            print("Links CSV downloaded successfully.")
```

Comment:
`print("Links CSV downloaded successfully.")` should probably be `print("{csv_type} CSV downloaded successfully.")`